build.gradle

```diff
@@ -8,3 +12,38 @@ dependencies {
 sourceCompatibility = '1.7'
 targetCompatibility = '1.7'
 
+wrapper.gradleVersion = '4.1.0'
+
+buildscript {
+    dependencies {
+        classpath files('libs/rsk-gradle-witness.jar')
+    }
+}
```

Comment:
We don't need these lines. The `wrapper.gradleVersion` is infered by `gradle-wrapper.properties` (I have to remove it from the `rskj-core` project, btw) and the whole `buildscript` block is for when you want to use this plugin, which in this case we don't